hikari/guilds.py

```diff
@@ -487,6 +498,15 @@ def mention(self) -> str:
         """
         return f"<@!{self.id}>" if self.nickname is not None else self.user.mention
 
+    def is_timed_out(self) -> bool:
+        """Whether the member is timed out.
+
+        !!! note
+            The output of this function can depend based on when
```

Comment:
This could probably just be `` Unlike `raw_communication_disabled_until`, this will never be in the past. `` if that other style was used.

Reply:
This returns a boolean tho

Reply:
Read the 2nd part of what I said

Reply:
Sorry, completely missed that